app/src/features/apiClient/screens/apiClient/APIClient.tsx

```diff
@@ -118,7 +118,7 @@ export const APIClient: React.FC<Props> = () => {
   const onNewClick = useCallback(() => {
     setSelectedEntry(getEmptyAPIEntry());
     setSelectedEntryDetails(null);
-    navigate(PATHS.API_CLIENT.ABSOLUTE);
+    navigate(`${PATHS.API_CLIENT.ABSOLUTE}/request/new`);
```

Comment:
Can we use the PATHS constants for new request path? if we don't have one then we can create like the `create` path for rule editors